.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Create a report to help us improve Zowe CLI
+labels: bug
+---
+
+<!-- Thanks for deciding to open an issue. Before submitting, please see the following information. -->
+
+<!-- Before opening a new issue, please search our existing issues: https://github.com/zowe/zowe-cli/issues -->
+
+<!--Also see the Troubleshooting section in the Zowe Docs website: https://docs.zowe.org/stable/troubleshoot/cli/known-cli
+-->
+
+**Describe the bug**
+
+<!-- A clear and concise description of the bug or error. -->
+
+**Steps to reproduce the bug**
+
+<!--
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
```

Comment:
These steps seem more suited to a graphical app like Zowe Explorer, perhaps we should change them to be more applicable to a Command Line Interface.

Not sure I have great ideas for what alternatives would be instead of "Go to", "Click on", "Scroll down", etc. 🤔 The only thing I can think of is:
```
<!--
Steps to reproduce the behavior:
1. Run command '...'
2. See error
-->
```